app/Phragile/SettingsAwareTransactionFilter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace Phragile;
+
+/***
```

Comment:
I think we use only two starts in this project? Or is this inconsistent anyways?